src/components/Policies/Policies.tsx

```diff
@@ -0,0 +1,48 @@
+import { useGameState } from "@/context/game-state/hook";
+import { PolicyCard as PolicyCardType } from "@/state/types";
+import { motion } from "framer-motion-3d";
+import { lowerXBoundary, lowerYBoundary, upperXBoundary, upperYBoundary } from "@/constants/gameBoard";
+import PolicyCard from "./PolicyCard";
+import { Html } from "@react-three/drei";
+import FundingCard from "./FundingCard";
+import { AnimatePresence } from "framer-motion";
+
+const Policies = () => {
+  const { state, emit, showPolicies, guards } = useGameState();
+
+  return (
+    <AnimatePresence>
+      {showPolicies && (
+        <motion.mesh key="policies" position={[0, 0, 8]} initial={{ y: -200 }} animate={{ y: 0 }} exit={{ y: -200 }}>
+          <planeGeometry args={[upperXBoundary - lowerXBoundary * 2, upperYBoundary - lowerYBoundary * 2, 1]} />
+          <meshBasicMaterial color="#052B4E" transparent opacity={0.9} />
+          <Html position={[0, 48, 1]} transform scale={7}>
+            <h1 className="font-light text-white">Active</h1>
+          </Html>
+          <Html position={[0, 15, 1]} transform scale={7}>
+            <h1 className="font-light text-white">Acquired</h1>
+          </Html>
+          <Html position={[0, -25, 1]} transform scale={7}>
+            <h1 className="font-light text-white">Funding</h1>
+          </Html>
```

Comment:
Why not use `<Text>` ?

Reply:
Text does not have the option to change the font without a custom font file so I used HTML for now